package/gluon-wireless-encryption/Makefile

```diff
@@ -0,0 +1,33 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-wireless-encryption
+PKG_VERSION:=1
+PKG_RELEASE:=1
+
+include ../gluon.mk
+
+define Package/gluon-wireless-encryption-wpa2
+  DEPENDS:=+hostapd-mini
+  TITLE:=Package for supporting WPA2 encrypted wireless networks
+endef
+
+define Package/gluon-wireless-encryption-wpa3
+  DEPENDS:=+hostapd-openssl
+  TITLE:=Package for supporting WPA3 encrypted wireless networks
+endef
+
+define Package/gluon-wireless-encryption-wpa2/install
+	$(INSTALL_DIR) $(1)/lib/gluon/features
+
+	touch $(1)/lib/gluon/features/wpa2
```

Comment:
Having a `wpa2` feature at all does not make sense to me; making it a part of the `wpa3` as well package even less.

Let's drop the flag and the package, and directly reference `hostapd-mini` in `package/features`.